README.md

```diff
@@ -40,6 +40,29 @@ Run the command (recommended):
 ...or to install from git, run:
 `cargo install --git https://github.com/altsem/gitu.git --locked`
 
+#### Using Release binaries
+
+gitu is available on Github Releases and should be installed from there.
+
+The latest release is available
+[here](https://github.com/altsem/gitu/releases).
+
+Download the archive that is appropriate for your platform and extract the
+binary into your `$PATH`. A common valid path location is `/usr/local/bin`.
+
+You can run these commands to automate this install on most platforms (*note:* it does not always work):
+
+```
+platform=$(uname -s)
+arch=$(uname -m)
+# If you're running on an M1 Macbook run this:
+# arch="x86_64"
+download_url=$(curl -s https://api.github.com/repos/altsem/gitu/releases/latest | grep "browser_download_url.*$arch.*${platform,,}" | cut -d : -f 2,3 | sed 's/"//g' | xargs)
+curl -L -o /tmp/gitu.tar.gz "$download_url"
+tar -C /tmp -xzvf /tmp/gitu.tar.gz
+mv $(find /$(readlink /tmp) -perm +111 -type f -name gitu 2>/dev/null) /usr/local/bin/
+```
+
```

Comment:
Perhaps this part about installing using a script is superfluous. I'd love to get publishing to package managers in the future though, like Homebrew! I'd keep the note about the `PATH` and `/usr/local/bin`

Reply:
Removed